esm-samples/rollup/rollup.config.js

```diff
@@ -15,7 +17,15 @@ export default {
     format: "es"
   },
   plugins: [
-    del({ targets: "public/chunks", runOnce: true, verbose: true }),
+    clean(),
+    del({ targets: ["public/chunks", "public/assets"], runOnce: true, verbose: true }),
+    copy({
+      // Copy the ArcGIS API for JavaScript assets
+      targets: [
+        { src: './node_modules/@arcgis/core/assets/*', dest: './public/assets'},
```

Comment:
- This can be simplified to `{ src: './node_modules/@arcgis/core/assets', dest: './public' },`
- It'd be good to use `prettier` to keep formatting consistent.

Reply:
I tried `./public` but it copies the files to `/public/esri`

Reply:
Did you change the `src` too?

Reply:
😄  Now it works.

Reply:
@dasa do you have any suggestions for the `prettier` rules?

Reply:
This is what we use for the API:

`.prettierrc.json`

```json
{
  "arrowParens": "always",
  "bracketSpacing": true,
  "endOfLine": "auto",
  "jsxBracketSameLine": false,
  "jsxSingleQuote": false,
  "printWidth": 120,
  "quoteProps": "consistent",
  "semi": true,
  "singleQuote": false,
  "tabWidth": 2,
  "trailingComma": "none",
  "useTabs": false,
  "overrides": [
    {
      "files": "*.tsx",
      "options": {
        "printWidth": 100
      }
    }
  ]
}
```

Maybe we could add this to the root of the repo so it applies to all the code? /cc @bsvensson

Reply:
I added the file to the repo's root